src/App.tsx

```diff
@@ -1,54 +1,115 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+
+const debounce = (
+  func: React.Dispatch<React.SetStateAction<string>>
+  | React.Dispatch<React.SetStateAction<boolean>>,
+  delay: number,
+) => {
+  let timer: number;
+
+  return (...args: any) => {
```

Comment:
Don't use any for types. Add the correct type here to fix lint warning (theoretically, the user should enter a string type there)
![image](https://github.com/mate-academy/react_autocomplete/assets/70009205/6c088796-78eb-43a9-9f3b-b52b4cbe2425)

Reply:
done